kerberos_client.go

```diff
@@ -42,10 +42,10 @@ func createClient(config *GSSAPIConfig, cfg *krb5config.Config) (KerberosClient,
 		if err != nil {
 			return nil, err
 		}
-		client = krb5client.NewClientWithKeytab(config.Username, config.Realm, kt, cfg)
+		client = krb5client.NewClientWithKeytab(config.Username, config.Realm, kt, cfg, krb5client.DisablePAFXFAST(config.DisablePAFXFAST))
```

Comment:
and now that you're modifying this file, could you please update the `NewKerberosClient` func comment to make it go friendly

![image](https://user-images.githubusercontent.com/950087/63527927-d8afc280-c4b6-11e9-9366-bba5f774fd16.png)

Reply:
Sure I'll do!